scripts/docker/Dockerfile.openmptarget

```diff
@@ -54,7 +54,7 @@ RUN LLVM_URL=https://github.com/llvm/llvm-project/archive &&\
       -DCMAKE_C_COMPILER=gcc \
       -DCMAKE_CXX_COMPILER=g++ \
       -DLLVM_ENABLE_PROJECTS="clang" \
-      -DLLVM_ENABLE_RUNTIMES="openmp" \
+      -DLLVM_ENABLE_RUNTIMES="openmp;libcxx;libcxxabi;libunwind" \
```

Comment:
Revert or describe why that would be needed.

Reply:
Dropped, not quite sure how that got there.